src/i18n.js

```diff
@@ -37,6 +41,10 @@ const resources = {
         'Je WiFi informatie wordt nooit naar de server verzonden. Tracking, analytics of fingerprinting wordt niet gebruikt op deze website. Bekijk de',
       'desc.source': 'broncode',
       'wifi.login': 'WiFi Login',
+      'wifi.size': 'Grootte',
+      'wifi.size.small': 'Klein',
+      'wifi.size.medium': 'Medium',
+      'wifi.size.large': 'Groot',
```

Comment:
@wouterbrink is it grammatically correct?

Reply:
Yep!

Reply:
Thanks!